### DISCUSSION TITLE
Reporting syntax highlighting bugs

### DISCUSSION THREAD

**Discussion by AdaDoom3** [community/community#21542] [New to GitHub]
<p>Hey there, what is the most effective way to get the devs to fix syntax highlighting issues? The Ada programming language’s highlighting continues to fluxuate in accuracy.</p>
<p>Just an example - notice that the “if” in “end if;” and the “loop” in “end loop;” are wrongly colored different than the rest of the if/loop keywords. Same for “case”… etc.</p>
<p>Also many things that should be differentiated are in fact left all black - like types and object names. :cry:</p>
<p><a href="https://github.com/AdaDoom3/AdaDoom3/blob/master/Engine/neo-data-model-idtech.adb" rel="nofollow noopener">https://github.com/AdaDoom3/AdaDoom3/blob/master/Engine/neo-data-model-idtech.adb</a></p>

**Comment by winstliu** [comment, marked as answer]
<p>Report it to <a href="https://github.com/textmate/ada.tmbundle" rel="nofollow noopener">https://github.com/textmate/ada.tmbundle</a>.</p>
<p>(Wondering how I knew?  Check out the list here: <a href="https://github.com/github/linguist/tree/master/vendor#grammar-index" rel="nofollow noopener">https://github.com/github/linguist/tree/master/vendor#grammar-index</a>)</p>

**Reply by OptimusPi** [reply]
Thank you!

**Reply by roke-julian-lockhart** [reply]
> https://github.com/orgs/community/discussions/21542#discussioncomment-3233634

@50Wliu, shouldn't reports go through https://github.com/github-linguist/linguist/issues/new?assignees=&labels=Bug&projects=&template=bug_report.md&title= first, generally?
